src/content/fr/2020/performance.md

```diff
@@ -0,0 +1,388 @@
+  )
+}}
+
+Le TTFB est affecté par la latence du réseau et le type de connexion. Plus la latence est élevée et plus la connexion est lente, plus les mesures de la TTFB sont mauvaises, comme nous pouvons l’observer ci-dessus. Même sur les connexions mobiles considérées comme rapides (4G), seuls 21 % des sites web ont un TTFB rapide. Il n’y a pratiquement aucun site classé comme rapide en dessous de la 4G.
+
+Si on examine [les vitesses de téléphonie mobile dans le monde pour la période décembre 2018-novembre 2019](https://www.speedtest.net/insights/blog/content/images/2020/02/Ookla_Mobile-Speeds-Poster_2020.png), on constate que, dans le monde, les connexions mobiles ne sont pas à haut débit. Ces vitesses et normes technologiques pour les réseaux cellulaires (comme la 5G) ne sont pas réparties de manière égale et affectent le TTFB. Par exemple, [voir cette carte des réseaux au Nigeria](https://www.mobilecoveragemaps.com/map_ng#7/8.744/7.670) - la plupart des régions du pays ont une couverture 2G et 3G, avec une faible portée en 4G.
+
+Ce qui est surprenant, c’est le nombre relativement identique de bons résultats de TTFB entre les sources hors ligne et 4G. En ce qui concerne les travailleurs du secteur des services, on pourrait s’attendre à ce que certains des problèmes liés au TTFB soient atténués, mais cette tendance ne se reflète pas dans le graphique ci-dessus.
+
+## Utilisation des <span lang="en">Performance Observers</span> {utilisation-des-performance-observers}
+
+Il existe des dizaines de mesures différentes, orientées utilisation, qui peuvent être utilisées pour évaluer les sites web et les applications. Cependant, il arrive que les mesures prédéfinies ne correspondent pas tout à fait à nos scénarios et besoins spécifiques. <a href="https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver" hreflang="en">L’API <span lang="en">PerformanceObserver</span></a> nous permet d’obtenir des données de mesure personnalisées grâce à <a href="https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API" hreflang="en">l’API <span lang="en">User Timing</span></a>, <a href="https://developer.mozilla.org/en-US/docs/Web/API/Long_Tasks_API" hreflang="en">l’API <span lang="en">Long Task</span></a>, <a href="https://web.dev/custom-metrics/#event-timing-api" hreflang="en">l’API <span lang="en">Event Timing</span></a> et [une poignée d’autres API de bas niveau](https://web.dev/custom-metrics/). Par exemple, avec leur aide, nous pourrions enregistrer les transitions temporelles entre les pages ou quantifier l’hydratation des applications en <span lang="en">Server Side Rendering</span> (SSR, rendues côté serveur).
+
+{{ figure_markup(
+  image="performance-performance-observer-usage.png",
+  caption="Usage des PerformanceObserver par type de matériel.",
+  description="Graphique en barres montrant que l’adoption de l’API PerformanceObserver est faible, à 6,6 % sur les ordinateurs de bureau et 7 % sur les téléphones portables.",
```

Comment:
```suggestion
  caption="Usage des Performance Observer par type de matériel.",
  description="Graphique en barres montrant que l’adoption de l’API Performance Observer est faible, à 6,6 % sur les ordinateurs de bureau et 7 % sur les téléphones portables.",
```

Reply:
I'm not sure about this. The API is really called PerformanceObserver (https://www.w3.org/TR/performance-timeline-2/#the-performanceobserver-interface)

Reply:
See your point, and didn't realise that, but reading that briefly I'm not sure I 100% agree anyway.

The interface is called `PerformanceObserver` but not sure I'd call the whole API that. The spec has no instance of "Performance Observer API" nor "PerformanceObserver API" and does talk about ["performance observers" and "performance observer tasks"](https://www.w3.org/TR/performance-timeline-2/#performance-timeline).

Even [web.dev seems confused](https://web.dev/custom-metrics/#performance-observer) calling it the "PerformanceObserver API" in some places and the "Performance Observer API" in others!

So I think there's arguments for both ways. Personally I'd go with spaces as no code formatting here. Also keeps it consistent with English.

WDYT?

Reply:
I don't think it matters that much. I'm ok with both or with mixing the two ways of writing it, so I'll accept the change.